src/lib/y2storage/clients/inst_prepdisk.rb

```diff
@@ -83,12 +89,38 @@ def mount_in_target(path, device, options)
         log.info "Cmd: mount #{options} #{device} #{target_path}"
 
         if !SCR.Execute(path(".target.mount"), [device, target_path], options)
-          raise ".target.mount failed"
+          # TRANSLATORS: %s is the path of a system mount like "/dev", "/proc", "/sys"
+          Yast::Report.Warning(_("Could not mount %s") % path )
         end
 
         nil
       end
 
+      # Check if efivars should be mounted, i.e. if /sys/firmware/efi/efivars
+      # exists and the system supports the efivarfs filesystem type.
+      #
+      # @return [Boolean] true if efivarfs should be mounted
+      def mount_efivars?
+        File.exist?(EFIVARS_PATH) && efivarfs_support?
+      end
+
+      # Check if the efivarfs filesystem type is supported on this system,
+      # i.e. if /proc/filesystems contains a line with "efivarfs".
+      #
+      # Notice that a system might have the /sys/firmware/efi/efivars file,
+      # but no support for the efivarfs filesystem to actually mount it.
+      # See https://bugzilla.suse.com/show_bug.cgi?id=1174029
+      #
+      # @return [Boolean] true if efivarfs is supported
+      def efivarfs_support?
+        begin
+          File.readlines("/proc/filesystems").any? { |line| line =~ /efivarfs/ }
+        rescue Errno::ENOENT => err
```

Comment:
Rubocop is (stupidly, IMO) going to complain because it wants this to be called `e`.